e2e/framework/README.md

```diff
@@ -0,0 +1,23 @@
+# rook-test-framework
+ 
+Requirements:
+-   Docker 1.12.6 
+-   Ubuntu Host
```

Comment:
Does it have to be Ubuntu or can it be any Linux host.  Do we have a minimum kernel version we support?